frontend/src/routes/ManageAdmissions/CreateAdmission.tsx

```diff
@@ -155,13 +179,22 @@ const CreateAdmission: React.FC = () => {
             toggleGroup={(value) => {
               formik.setFieldValue(
                 "groups",
-                toggleFromArray<Group>(formik.values.groups, value)
+                toggleFromArray(formik.values.groups, value)
               );
             }}
           />
         </InputWrapper>
       </FormGroup>
-      <LegoButton type="submit">Opprett opptak</LegoButton>
+      {returnedData && (
+        <FormGroup>
+          <ResultText type={returnedData.type}>
+            {returnedData.message}
+          </ResultText>
+        </FormGroup>
+      )}
```

Comment:
I don't _really_ like this error handling stuff with the state. This really feels like it should be some global snackbar or something similar and not handled here.

 But if you don't want to change it now that's fine.

Reply:
True, that would be a lot nicer to have, both here and when deleting an application. Adding a new issue to take a look at that.